shopfloor_base/models/shopfloor_app.py

```diff
@@ -281,6 +285,13 @@ def _is_component_registry_ready(self):
         return comp_registry and comp_registry.ready
 
     def _get_services(self):
+        forced_services = self.env.context.get("sf_service_components")
+        if forced_services:
+            _logger.debug(
+                "_get_services forced services: %s",
+                ", ".join([x._usage for x in forced_services]),
+            )
+            return forced_services
```

Comment:
shouldn't we add those to the regular result?

Reply:
There's no result when you use this (that's why we use it :wink: ).